src/phongo_ini.c

```diff
@@ -14,96 +14,82 @@
  * limitations under the License.
  */
 
-#include "bson/bson.h"
-#include "mongoc/mongoc.h"
+#include <stdio.h>
 
 #include <php.h>
 #include <main/php_open_temporary_file.h>
-#include <ext/date/php_date.h>
 
 #include "php_phongo.h"
 #include "phongo_ini.h"
+#include "phongo_log.h"
 
 ZEND_EXTERN_MODULE_GLOBALS(mongodb)
 
-static void phongo_log(mongoc_log_level_t log_level, const char* log_domain, const char* message, void* user_data)
+static FILE* phongo_ini_tmp_file(const char* tmp_dir)
 {
-	struct timeval tv;
-	time_t         t;
-	zend_long      tu;
-	zend_string*   dt;
+	time_t       t;
+	int          fd = -1;
+	char*        prefix;
+	int          len;
+	zend_string* filename = NULL;
+	FILE*        stream   = NULL;
 
-	bson_gettimeofday(&tv);
-	t  = tv.tv_sec;
-	tu = tv.tv_usec;
+	/* TODO: consider using time() retval directly instead of time_t struct */
```

Comment:
I do intend to come back to this before merging. I just want to double-check with the C team that I'm not inviting compatibility issues by moving away from `bson_gettimeofday()`. One would _hope_ that [time(2)](https://man7.org/linux/man-pages/man2/time.2.html) is old enough to rely on across platforms.

Reply:
Replaced this with `time(NULL)`, which is already used in PHP core: https://github.com/php/php-src/blob/php-8.2.10/ext/date/php_date.c#L70